src/video/SDL_video.c

```diff
@@ -2223,9 +2223,9 @@ void SDL_SetWindowBordered(SDL_Window *window, SDL_bool bordered)
 void SDL_SetWindowResizable(SDL_Window *window, SDL_bool resizable)
 {
     CHECK_WINDOW_MAGIC(window, );
-    if ((window->flags & SDL_WINDOW_FULLSCREEN) == 0) {
+    if (!(window->flags & SDL_WINDOW_FULLSCREEN)) {
         const int want = (resizable != SDL_FALSE); /* normalize the flag. */
-        const int have = ((window->flags & SDL_WINDOW_RESIZABLE) != 0);
+        const int have = (window->flags & SDL_WINDOW_RESIZABLE);
```

Comment:
This is actually comparing int values as boolean, so this probably needs to be switched to SDL_bool or left alone.